llama_index/vector_stores/chroma.py

```diff
@@ -99,6 +100,7 @@ def __init__(
             port=port,
             ssl=ssl,
             headers=headers,
+            persist_dir=persist_dir,
```

Comment:
doesn't make sense to pass this here since `persist_dir` is not a field.

Reply:
@Disiok this has been now added as a field. I think we're good to resolve this comment.